docs/src/man/split_apply_combine.md

```diff
@@ -122,6 +122,10 @@ It is allowed to mix single values and vectors if multiple transformations
 are requested. In this case single value will be repeated to match the length
 of columns specified by returned vectors.
 
+A separate task is spawned for each specified transformation, allowing for
+parallel operation when several transformations are requested and Julia was
+started with more than one thread.
```

Comment:
Maybe add that this means that the transformation functions passed should not modify the same state of the Julia program?

Reply:
Good idea. I've also mentioned that this may be extended in the future so that people are not caught by surprise.